frontend/store.js

```diff
@@ -13,23 +13,31 @@ const reducer = combineReducers({
   routing: routerReducer,
 });
 
-const middlewares = [
-  reroute,
-  createNotifier(notificationSettings),
-  createLogger(), // must be last in the middlewares chain
-];
-
 // FRONTEND_CONFIG is a global variable rendered into the index
 // template by the Main controller. We use it to initialize our
 // store's state with configuration values from the server-side.
 const FRONTEND_CONFIG = typeof window !== 'undefined'
   ? window.FRONTEND_CONFIG
   : global.FRONTEND_CONFIG;
 
+const middlewares = [
+  reroute,
+  createNotifier(notificationSettings),
+];
+
+const enhancers = [
+  applyMiddleware,
+];
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
The production webpack build runs with `-p` which adds `process.env.NODE_ENV="production"`. In development, this will be `null` or `undefined`.